modules/lambdas/main.tf

```diff
@@ -456,6 +461,8 @@ resource "aws_lambda_function" "db_deploy" {
 data "aws_lambda_invocation" "db_migration" {
   depends_on    = [aws_lambda_function.db_deploy]
   function_name = aws_lambda_function.db_deploy.function_name
-  input         = jsonencode({})
+  input         = jsonencode({
+    replacementTrigger = timestamp()
+  })
```

Comment:
This should prevent the lambda invocation from running during the `terraform plan` phase and only run it during the `terraform apply` phase. This is important because otherwise if you have somehow misconfigured your deployment, subsequent deployments will keep running this Lambda invocation with your old configuration before your new configuration can be deployed, creating an endless loop that you can't get out of.